File: frontend/src/router.ts
import { createRouter, createWebHistory } from 'vue-router'

const router = createRouter({
  history: createWebHistory(),
  routes: [
    {
      path: '/',
      redirect: '/lessons',
    },
    {
      path: '/lessons',
      name: 'lessons',
      component: () => import('@/views/LessonsView.vue'),
    },
    {
      path: '/students',
      name: 'students',
      component: () => import('@/views/StudentsView.vue'),
    },
    {
      path: '/subjects',
      name: 'subjects',
      component: () => import('@/views/SubjectsView.vue'),
    },
    {
      path: '/analytics',
      name: 'analytics',
      component: () => import('@/views/AnalyticsView.vue'),
    },
  ],
})

export default router

File: backend/app/api/router.py
from fastapi import APIRouter

from app.api.routes.analytics import router as analytics_router
from app.api.routes.lessons import router as lessons_router
from app.api.routes.students import router as students_router
from app.api.routes.subjects import router as subjects_router

api_router = APIRouter()
api_router.include_router(students_router)
api_router.include_router(subjects_router)
api_router.include_router(lessons_router)
api_router.include_router(analytics_router)
